diagnostics/LampStates lamp_states
diagnostics/LampDriverStates lamp_driver_states
std_msgs/Bool ok